src/main/java/task/Task.java

```diff
@@ -0,0 +1,39 @@
+package task;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description){
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public void setDescription(String description){
+        this.description = description;
+        return;
```

Comment:
`void` functions do not need a `return` statement unless you are preliminarily exiting the function.